### DISCUSSION TITLE
2021-07-15 Dev Platform

### DISCUSSION THREAD

**Discussion by mikedotexe** [near/community#74] [Archived]
Updates related to Dev Platform team: https://github.com/near/dev-platform
Rust-related board: https://github.com/orgs/near/projects/10
Tooling/libraries board: https://github.com/orgs/near/projects/13

## Done
**Non-Rust**
- Bo, Josh Ford, Mike and others reviewed and merged the (somewhat alpha approach) Sandbox instructions to docs. That now lives at: https://docs.near.org/docs/develop/contracts/sandbox
- Sandbox implementation merged for the Rust Status Message example at: https://examples.near.org/rust-status-message
- Serhii increasing the NEAR CLI default gas to max. https://github.com/near/near-cli/commit/62b2566033a32ab9846168c25010e2cdf04e55c0
- Serhii published releases of `near-api-js` and NEAR CLI. https://github.com/near/near-api-js/releases/tag/v0.42.0 and https://github.com/near/near-cli/releases/tag/v2.1.0. Serhii also increased his age by `1u8`.
- Multi-token standard let by Zane Starr went well, meeting notes here: https://github.com/shipsgold/multi-token-standard-impl/blob/main/meetings/2021-07-07-meeting-agenda.md

**Rust**
- Phuong's PRs merged https://github.com/near/near-sdk-rs/pull/469 and https://github.com/near/near-sdk-rs/pull/464
- Austin cleaned up some examples to reflect the new, less-boilerplate feature where the global memory allocator doesn't need to be added each time: https://github.com/near/near-sdk-rs/pull/459
- Mike wrote instructions for the simulation tests for the Rust SDK Docs, as they'll be needed in the interim before the Sandbox. Note, this link will change when our domain is secured soon: https://sdk-g4yv.onrender.com/testing/simulation-tests
- Also for Rust SDK Docs, Austin published this section https://github.com/near/sdk-docs/pull/30
- Austin improving the `borsh-rs` repo with: https://github.com/near/borsh-rs/commit/88459fcab4e1db55bf20c1c50d2fb414b8df3271

## In Progress
- Austin has been exploring the approach used by `near-sdk-rs` and is fleshing out ideas for contract size / gas reduction in a project he's calling nesdie: https://github.com/austinabell/nesdie 
  This will be informing some decisions around the OKRs about reduction and efficiency of the SDK as we meet with leadership and others about the pain points and approaches that have worked.
- Josh Quintal and Serhii will published the public version of the NEAR CLI and library proposal to the governance site.

**Comment by maxzaver** [comment]
It seems like @vgrichina is still publishing near-api-js. I suggest DevPlatform takes publishing of the releases over.

**Reply by mikedotexe** [reply]
Got it, thank you
